bedrock/security/templates/security/bug-bounty/web-eligible-sites.html

```diff
@@ -0,0 +1,190 @@
+      </h2>
+
+      <p>These websites and services are considered critical to Mozilla operations and pay out at the highest bounty rate.</p>
+      <h4>ABSearch</h4>
+      <ul>
+        <li>search.services.mozilla.com</li>
+      </ul>
+
+      <h4>Add-ons</h4>
+      <ul>
+        <li>addons.cdn.mozilla.net</li>
+        <li>addons.mozilla.org</li>
+        <li>blocklist.addons.mozilla.org</li>
+        <li>builder.addons.mozilla.org</li>
+        <li>compatibility-lookup.services.mozilla.com</li>
+        <li>discovery.addons.mozilla.org</li>
+        <li>services.addons.mozilla.org</li>
+        <li>static.addons.mozilla.net</li>
+        <li>versioncheck-bg.addons.mozilla.org</li>
+        <li>versioncheck.addons.mozilla.org</li>
+      </ul>
+
+      <h4>Bedrock (www)</h4>
+      <ul>
+        <li>www.firefox.com</li>
+        <li>www.getfirefox.com</li>
+      	<li>www.mozilla.com</li>
+      	<li>www.mozilla.org</li>
+      </ul>
+
+      <h4>Bugzilla</h4>
+      <ul>
+        <li>bugzilla.mozilla.org</li>
+        <li>*.bmoattachments.org (excludes XSS and open redirects)</li>
+      </ul>
+      <p>
+        <strong>Please do not use automated scanners, create, or modify bugs when testing Bugzilla.</strong>
+        <br><br>Instead, install your own local copy for testing from <a href="https://github.com/mozilla/webtools-bmo-bugzilla">webtools-bmo-bugzilla</a> or use our <a href="https://bugzilla-dev.allizom.org/">development instance</a>.
+      </p>
+
+      <h4>Crash Reports</h4>
+      <ul>
+        <li>crash-reports.mozilla.com</li>
+        <li>crash-reports-xpsp2.mozilla.com</li>
+        <li>crash-stats.mozilla.com</li>
+      </ul>
+
+      <h4>Downloads (Product Delivery)</h4>
+      <ul>
+        <li>archive.mozilla.org</li>
+        <li>download.mozilla.org</li>
+      </ul>
+
+      <h4>Firefox Accounts</h4>
+      <ul>
+        <li>accounts.firefox.com</li>
+        <li>api.accounts.firefox.com</li>
+        <li>oauth.accounts.firefox.com</li>
+        <li>profile.accounts.firefox.com</li>
+        <li>verifier.accounts.firefox.com</li>
+      </ul>
+
+      <h4>Firefox Settings (Kinto)</h4>
+      <ul>
+        <li>firefox.settings.services.mozilla.com</li>
+        <li>webextensions.settings.services.mozilla.com</li>
+      </ul>
+
+      <h4>Firefox Sync</h4>
+      <ul>
+        <li>*.sync.services.mozilla.com</li>
+        <li>token.services.mozilla.com</li>
+      </ul>
+
+      <h4>Firefox Updates (AUS/Balrug)</h4>
+      <ul>
+        <li>aus3.mozilla.org</li>
+        <li>aus4.mozilla.org</li>
+        <li>aus5.mozilla.org</li>
+      </ul>
+
+      <h4>Push</h4>
+      <ul>
+        <li>push.services.mozilla.com</li>
+        <li>updates.push.services.mozilla.com</li>
+      </ul>
+
+      <h4>Shield</h4>
+      <ul>
+        <li>self-repair.mozilla.org</li>
+      </ul>
+
+      <h4>Source Control</h4>
+      <ul>
+        <li>hg.mozilla.org (excludes website)</li>
```

Comment:
Can this say '(except website, see below)'? I initially interpreted it as a flat denial for https://hg and didn't think "oh, they re-include that later on" at first read.